ConsoleApp/Models/Circle.cs

```diff
@@ -0,0 +1,33 @@
+namespace ConsoleApp.Models
+{
+    public class Circle : Figure
+    {
+        public int Radius { get; set; }
+        public override double Square=>Math.PI*Radius*Radius;
+        public override double Perimeter => 2*Math.PI*Radius;
+
+        public Circle(int radius)
+        {
+            Radius = Math.Abs(radius);
```

Comment:
if I pass negative radius, exception should be thrown, `Abs` is bad practice in this case